allennlp/data/dataset_readers/dataset_utils/ontonotes.py

```diff
@@ -64,7 +64,7 @@ def __init__(self,
                  word_senses: List[Optional[float]],
                  speakers: List[Optional[str]],
                  named_entities: List[str],
-                 srl_frames: Dict[str, List[str]],
```

Comment:
Just curious, what's the motivation for switching from a Dict to a list of tuples?

Reply:
Oh, I see the reason now (having read further down). Because it seemed like the cleanest way to allow duplicates of the key (as opposed to making keys map to lists of instances). That makes sense to me.